ajax/libs/upng-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "upng-js",
+  "description": "Small, fast and advanced PNG / APNG encoder and decoder",
+  "version": "2.0.0",
+  "author": "photopea",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/photopea/UPNG.js"
+  },
+  "filename": "UPNG.min.js",
```

Comment:
Let's move this filed between `name` and `description`.
Let people to saw this at the first glance.
This will useful to people to read.

Thanks.